go.mod

```diff
@@ -22,7 +22,7 @@ retract (
 
 require (
 	cel.dev/expr v0.15.0 // indirect
-	code.cloudfoundry.org/clock v1.0.0 // indirect
+	code.cloudfoundry.org/clock v1.21.0 // indirect
```

Comment:
I'd drop the `go.mod` and `go.sum` changes from this. We're not changing any dependencies here, and the upgrades should be handled by dependabot. That'll make the PR simpler.